tests/testServerWithStorage.js

```diff
@@ -487,6 +487,52 @@ sjrk.storyTelling.server.testServerWithStorageDefs.testImageRetrieval = function
     jqUnit.assertEquals("header.content-length is 3719", "3719", request.nativeResponse.headers["content-length"]);
 };
 
+jqUnit.test("Test isValidMediaFilename function", function () {
+    jqUnit.expect(15);
+
+    var inputFileNames = [
```

Comment:
I think this would be clearer if we combined the test cases and expected value in a single array.